conf/app.routes

```diff
@@ -128,16 +128,30 @@ GET        /import-vat-overpayment                    controllers.VATPaidControl
 POST       /import-vat-overpayment                         controllers.VATPaidController.onSubmit(mode: Mode = NormalMode)
 GET        /change-import-vat-overpayment                  controllers.VATPaidController.onPageLoad(mode: Mode = CheckMode)
 POST       /change-import-vat-overpayment                  controllers.VATPaidController.onSubmit(mode: Mode = CheckMode)
+GET        /repay-import-vat-overpayment                  controllers.VATPaidController.onPageLoad(mode: Mode = RepayNormalMode)
```

Comment:
I am just thinking can you get rid of RepayNormal and RepayCheckMode all together. I have a feeling you can achieve the desired functionality with CheckMode and NormalMode

Reply:
I tried, but not working for all the scenarios. We can pair up tomorrow if you are ok.

Reply:
yes sure